conda-store-server/conda_store_server/_internal/worker/app.py

```diff
@@ -72,10 +72,19 @@ def initialize(self, *args, **kwargs):
         # ensure checks on redis_url
         self.conda_store.redis_url
 
+    def logger_to_celery_logging_level(self, logging_level):
+        # celery supports the log levels DEBUG | INFO | WARNING | ERROR | CRITICAL | FATAL
+        # https://docs.celeryq.dev/en/main/reference/cli.html#celery-worker
+        logging_to_celery_level_map = {
+            50:"CRITICAL", 50:"FATAL", 40: "ERROR", 30: "WARNING", 20: "INFO", 10: "DEBUG"
```

Comment:
There isn't really a 1:1 relationship between the logging package error levels and the expected logging levels for celery. So, I think maintaining a map is required. Definitely open to suggestions if there is a better place to put this, like maybe a utils type file that I missed.